Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -277,7 +277,7 @@ - (void)_setupVisibilityPropagationViewForWebProcess
     if (!_visibilityPropagationInteractionForWebProcess) {
         SEL selector = NSSelectorFromString(@"createVisibilityPropagationInteraction");
         if ([_page->process().extensionProcess().get() respondsToSelector:selector])
-            _visibilityPropagationInteractionForWebProcess = adoptNS([_page->process().extensionProcess() performSelector:selector]);
+            _visibilityPropagationInteractionForWebProcess = [_page->process().extensionProcess() performSelector:selector];
```

Comment:
Oops! Good catch.

Reply:
> Weird for something named `createVisibilityPropagationInteraction` to not return an object that needs adopting...

That is a good point. I think I will land as-is, but will look closer into this.

Thanks for reviewing!